trikScriptRunner/src/pythonEngineWorker.cpp

```diff
@@ -152,11 +152,17 @@ bool PythonEngineWorker::evalSystemPy()
 		QLOG_ERROR() << "system.py not found, path:" << systemPyPath;
 		return false;
 	}
-	mMainContext.evalFile(systemPyPath);
+
+	mMainContext.evalScript("script.wait(0)");
```

Comment:
```suggestion
	// HACK: to avoid duplicate system.py try to check if basic feature like script.wait works.
	mMainContext.evalScript("script.wait(0)");
```